calliope/backend/pyomo/constraints/__init__.py

```diff
@@ -0,0 +1,12 @@
+from calliope.backend.pyomo.constraints.capacity import *
```

Comment:
Maybe we should put all constraint functions into one file so these import statements aren't needed -- these files no longer work as one would expect Python modules to work, since all their contents are dumped into one namespace here anyway. @timtroendle? @brynpickering? And perhaps this should be called "rules" rather than "constraints", since it also contains rules for expressions.

Reply:
It's my usual refrain, but I wouldn't concern ourselves with these details for now: with custom constraints implemented we'll be able to re-write these constraints in YAML and can do away with all these files anyway.

Reply:
Ok, let's leave it be for now

Reply:
Agree with Bryn that it might not be the right moment to do so.

If we were to keep these files for a longer time I'd suggest some sort of a callback mechanism that inverts the current call order. There could be a registry of whatever form in this file here (the simplest being a Python list) into which constraints (or constraint module) add themselves: "Hello, I am a constraint. Call me back whenever you needs constraints."